tsdb/engine/tsm1/compact.go

```diff
@@ -21,15 +21,22 @@ import (
 	"time"
 )
 
-// maxCompactionSegments is the maximum number of segments that can be
-// compaction at one time.  A lower value would shorten
+// minCompactionFileCount is the minimum number of TSM files that need to
+// exist before a compaction cycle will run
+const minCompactionFileCount = 5
+
+// maxCompactionFileCount is the maximum number of TSM files that can be
+// compacted at one time.  A lower value would shorten
 // compaction times and memory requirements, but produce more TSM files
 // with lower compression ratios.  A higher value increases compaction times
-// and memory usage but produces more dense TSM files.
-const maxCompactionSegments = 10
+// and memory usage but produces more dense TSM files. This value is a cutoff
```

Comment:
You're right.  Comment is incorrect because it was rolled over from when these were WAL segments.